### ISSUE TITLE
source code

### ISSUE THREAD

**Issue by XIAsher** [AsterqlDev/Pizza-Client#3]
can you please release the source code of the mod so i can check that its not a rat or a session stealer?

**Comment by AliEolo** [comment]
did you check?

**Comment by AsterqlDev** [comment]
Yes I can add source code. I am in process of updating mod so it will be up soon.
